.gitignore

```diff
@@ -24,3 +24,7 @@
 /.node_modules.ember-try/
 /bower.json.ember-try
 /package.json.ember-try
+
+
+ 
+package-lock.json
```

Comment:
shouldn't it be /package-lock.json, package.lock is pushed with the code also as I can see.

Reply:
I have added `package-lock` in the gitignore later so that is why it is showing in the code.
Both `/package.lock` and `package.lock.` will work.

Reply:
so can you please revert it?

Reply:
@vinayak-trivedi @manish591  The file is deleted that's why it's in the PR as file changed
![image](https://user-images.githubusercontent.com/78433013/185549423-09a4759c-0fb9-48ae-8639-462648fc8563.png)

Reply:
Adding a slash would be good to `package-lock.json` but only the file name will also work
![image](https://user-images.githubusercontent.com/78433013/185551465-3c788d49-f43a-4039-aaa8-e048e6ad9a00.png)
https://www.atlassian.com/git/tutorials/saving-changes/gitignore